src/components/atoms/Potrait.tsx

```diff
@@ -0,0 +1,63 @@
+import { PropsWithChildren, useEffect, useState } from "react";
+import styled from "styled-components";
+import Icons from "./Icons";
+
+type Props = PropsWithChildren<
+    React.DetailedHTMLProps<
+        React.HTMLAttributes<HTMLImageElement>,
+        HTMLImageElement
+    > & { src?: string }
+>;
+
+const Container = styled.div`
+    width: 18.75em;
+    position: relative;
+    overflow: hidden;
+    border-radius: 0.4em;
+    aspect-ratio: 14 / 16;
+
+    img {
+        width: 100%;
+        height: 100%;
+        object-fit: cover;
+        position: relative;
+    }
+
+    .loading {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        position: absolute;
+        inset: 0;
+        width: 100%;
+        height: 100%;
+        background-color: rgba(45, 116, 255, 0.5);
+    }
+`;
+
+const Potrait = ({ onLoad, ...props }: Props) => {
```

Comment:
```suggestion
const Portrait = ({ onLoad, ...props }: Props) => {
```

Talvez aqui seria dessa forma?